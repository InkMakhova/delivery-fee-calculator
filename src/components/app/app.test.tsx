import App from './app';
import { render } from "@testing-library/react";

it('should render App correctly', () => {
  const {container} = render(<App />);

  expect(container).toMatchSnapshot();
});
