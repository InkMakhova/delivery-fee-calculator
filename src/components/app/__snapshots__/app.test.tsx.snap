// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App correctly 1`] = `
<div>
  <div
    style="width: 50%; margin: 1em auto;"
  >
    <form>
      <fieldset>
        <legend>
          Delivery fee calculator
        </legend>
        <div
          class="form-group"
          style="margin-bottom: 1em; display: flex; justify-content: space-between; align-items: center;"
        >
          <label
            class="col-sm-2 col-form-label"
            for="value"
            style="margin-right: 1em; padding: 0px;"
          >
            Cart value
          </label>
          <div
            class="col-sm-10"
            style="margin-right: 1em;"
          >
            <input
              aria-describedby="cart-value"
              class="form-control"
              id="value"
              placeholder="Cart value"
              required=""
              type="text"
              value=""
            />
          </div>
          <p>
            €
          </p>
        </div>
        <div
          class="form-group"
          style="margin-bottom: 1em; display: flex; justify-content: space-between; align-items: center;"
        >
          <label
            class="col-sm-2 col-form-label"
            for="distance"
            style="margin-right: 1em; padding: 0px;"
          >
            Delivery distance
          </label>
          <div
            class="col-sm-10"
            style="margin-right: 1em;"
          >
            <input
              class="form-control"
              id="distance"
              placeholder="Delivery distance"
              required=""
              type="text"
              value=""
            />
          </div>
          <p>
            m
          </p>
        </div>
        <div
          class="form-group"
          style="margin-bottom: 1em; display: flex; justify-content: space-between; align-items: center;"
        >
          <label
            class="col-sm-2 col-form-label"
            for="number"
            style="margin-right: 1em; padding: 0px;"
          >
            Number of items
          </label>
          <div
            class="col-sm-10"
            style="margin-right: 1em;"
          >
            <input
              class="form-control"
              id="number"
              placeholder="Number of items"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="form-group"
          style="margin-bottom: 1em; display: flex; justify-content: space-between; align-items: center;"
        >
          <label
            class="col-sm-2 col-form-label"
            for="time"
            style="margin-right: 1em; padding: 0px;"
          >
            Order time
          </label>
          <div
            class="col-sm-10"
            style="margin-right: 1em;"
          >
            <input
              class="flatpickr-input"
              data-enable-time="true"
              readonly="readonly"
              type="text"
              value=""
            />
          </div>
        </div>
        <button
          class="btn btn-primary"
          style="margin-bottom: 1em; margin-top: 1em; margin-right: 1em;"
          type="submit"
        >
          Calculate delivery price
        </button>
        <button
          style="background-color: transparent; color: blue; text-decoration: underline;"
          type="reset"
        >
          Reset form
        </button>
      </fieldset>
    </form>
    <p
      style="margin-right: 1em; font-size: 1.5em;"
    >
      Delivery price: 
      <span
        style="font-size: 1em;"
      >
        0
         
      </span>
      €
    </p>
  </div>
</div>
`;
